lib/xpm_ruby/category.rb

```diff
@@ -0,0 +1,17 @@
+require "ox"
+
+module XpmRuby
+  module Category
+    extend self
+
+    class Error < Error; end
```

Comment:
```suggestion
    class Error < StandardError; end
```

Reply:
Actually why do we need to define this here ? I noticed other modules also have this.

Reply:
yeah, I think it might be a severe case of 🍝

Reply:
@abreckner might be able to answer. I'll say just delete it if we don't use it.